rbi/annotations/activerecord.rbi

```diff
@@ -196,6 +196,12 @@ class ActiveRecord::Base
 end
 
 class ActiveRecord::Relation
+  Elem = type_member(:out) { { fixed: T.untyped } }
```

Comment:
Hmmm, it's odd that Sorbet is requiring this, as `ActiveRecord::Relation` already does an `include Enumerable`:

https://github.com/rails/rails/blob/f481602bc5f0e2b79f95b73d7cc343303dd69672/activerecord/lib/active_record/relation.rb#L67